Dockerfile.openshift

```diff
@@ -43,12 +43,13 @@ ENV PYTHONDONTWRITEBYTECODE yes
 RUN INSTALL_PKGS=" \
 	tcpdump libpcap \
 	iproute iputils strace socat \
-	frr \
 	python3 \
 	podman-catatonit" && \
 	yum install -y --setopt=tsflags=nodocs --setopt=skip_missing_names_on_install=False $INSTALL_PKGS
 
-RUN dnf -y update && yum clean all && rm -rf /var/cache/yum/* && rm -rf /var/cache/yum
+RUN dnf -y update && \
+yum install -y --setopt=tsflags=nodocs --setopt=skip_missing_names_on_install=False frr-8.5.3-4.el9 && \
```

Comment:
The switching between yum and dnf here is necessary? It's not new so I'll lgtm regardless, just curious.

Reply:
it was using yum to install frr before, so kept it the same (as you mentioned). we should maybe switch them all to dnf in a different pr 😅